src/model/add-dots.ts

```diff
@@ -0,0 +1,60 @@
+// Check if the block contains a dot, replace with an empty span with a bullet attribute
+const hasDots = (element: CAPIElement): boolean => {
+	if (element._type !== 'model.dotcomrendering.pageElements.TextBlockElement')
+		return false;
+
+	if (element.html.includes('•')) {
+		element.html = element.html.replace(
+			new RegExp('[•]', 'g'),
+			'<span data-dcr-style="bullet"></span>',
+		);
+		return true;
+	}
+
+	return false;
+};
+
+const checkForDots = (elements: CAPIElement[]): CAPIElement[] => {
+	// Loop over elements and check if a dot is in the TextBlockElement
+	const enhanced: CAPIElement[] = [];
+	elements.forEach((element) => {
+		if (
+			element._type ===
+				'model.dotcomrendering.pageElements.TextBlockElement' &&
+			hasDots(element)
```

Comment:
am I right in thinking this works because `hasDots` is side-effecting? I think now that we're explicitly changing the element in `hasDots` we should call it `transformDots` and have a return type of Element.